### ISSUE TITLE
Multiline template not working on Windows

### ISSUE THREAD

**Issue by franzbischoff** [mschubert/clustermq#238]
https://github.com/mschubert/clustermq/blob/34e05ace1265a6dd9724df716d9376496608addb/inst/SSH.tmpl#L1

After several attempts, `Remote R process did not respond after 5000 seconds.` and no logs, I found that ssh command was complaining.

So I created a copy of your template to use as a custom template, and just removed the newlines `\`. And it connects.

**Comment by mschubert** [comment]
Thank you for finding the issue & the solution!

I'll have to think of a good way to include ssh with CI on Windows/Mac to avoid this in the future.

**Comment by franzbischoff** [comment]
Anyways, I tried again using ssh inside the WSL2 itself, because while between windows and WSL2 it "connects", there is no progression. I tried to disable the firewall, but I still think is something about the port forwarding that is different in WSL2 than it was in WSL.

In resume: win-to-WSL2, the above-mentioned hack works to connect the workers, but I don't receive any return, forever.
WSL2 to WSL2, works fine.

**Comment by mschubert** [comment]
Yes, that's not a surprise because Windows didn't (doesn't?) allow SSH reverse tunneling for a very long time.

In any case, we should error out cleanly instead of hanging.
